sqlness/src/interceptor/sleep.rs

```diff
@@ -74,7 +75,9 @@ impl InterceptorFactory for SleepInterceptorFactory {
                 prefix: PREFIX.to_string(),
                 msg: format!("Failed to parse milliseconds: {}", e),
             })?;
-        Ok(Box::new(SleepInterceptor { milliseconds }))
+        Ok(Box::new(SleepInterceptor {
+            duration: Duration::from_millis(milliseconds),
```

Comment:
This is not what I mean, I expect the params used in template is duration, like `-- SQLNESS SLEEP 15s`.